lib/special_route_publisher.rb

```diff
@@ -1,10 +1,25 @@
+require 'gds_api/publishing_api'
 require 'gds_api/publishing_api/special_route_publisher'
 
 class SpecialRoutePublisher
   def initialize(publisher_options)
     @publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new(publisher_options)
   end
 
+  def take_ownership_of_search_routes
+    publishing_api_v1 = GdsApi::PublishingApi.new(
```

Comment:
What does `v1` mean here? I'm a bit hazy on the publishing API's history, but I thought everything used v2 by default?

Reply:
I'm also unfamiliar with the v1 API. Is this functionality unavailable in v2?